rpcs3/Emu/Audio/AudioBackend.cpp

```diff
@@ -100,13 +100,13 @@ void AudioBackend::normalize(u32 sample_cnt, const f32* src, f32* dst)
 	}
 }
 
-std::pair<AudioChannelCnt, AudioChannelCnt> AudioBackend::get_channel_count_and_downmixer(u32 device_index)
+std::pair <AudioChannelCnt, AudioChannelCnt> AudioBackend::get_channel_count_and_downmix(u32 device_index)
 {
 	audio_out_configuration& audio_out_cfg = g_fxo->get<audio_out_configuration>();
 	std::lock_guard lock(audio_out_cfg.mtx);
 	ensure(device_index < audio_out_cfg.out.size());
 	const audio_out_configuration::audio_out& out = audio_out_cfg.out.at(device_index);
-	return out.get_channel_count_and_downmixer();
+	return out.get_channel_count_and_downmix();
```

Comment:
this is all unnessecary :)

Reply:
Oh whoops spelled it wrong

Reply:
you're not only changing history for no reason you also introduce conflicts with the PR right below yours by renaming the thing